docs/ref/validators.txt

```diff
@@ -222,12 +222,13 @@ to, or in lieu of custom ``field.clean()`` methods.
 ``int_list_validator``
 ----------------------
 
-.. function:: int_list_validator(sep=',', message=None, code='invalid')
+.. function:: int_list_validator(sep=',', message=None, code='invalid', allow_negative=False)
 
     .. versionadded:: 1.9
 
     Returns a :class:`RegexValidator` instance that ensures a string
-    consists of integers separated by ``sep``.
+    consists of integers separated by ``sep``
```

Comment:
. It allows.... (wrap to previous line)
Also need a `.. versionchanged:: 1.10` annotation and a mention in the 1.10 release notes.